service/src/main/resources/db/migration/V218__invoice_correction.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE invoice_correction (
+    id uuid PRIMARY KEY DEFAULT ext.uuid_generate_v1mc(),
+    created timestamp with time zone DEFAULT now() NOT NULL,
+    updated timestamp with time zone DEFAULT now() NOT NULL,
+    head_of_family_id uuid NOT NULL REFERENCES person(id),
```

Comment:
Should there be an index on head_of_family?

Reply:
Yeah, definitely 👍